www/runtest.php

```diff
@@ -2011,6 +2011,17 @@ function CreateTest(&$test, $url, $batch = 0, $batch_locations = 0)
                 $testFile .= "customMetric=$name:$code\r\n";
             }
 
+            // Add Custom Headers
+            if ($test['script'] && isset($test['webdriver']) && $test['webdriver']) {

```

Comment:
Looks like we could even keep it on for non webdriver mode, right? This way we could remove the section bellow and avoid duplicating code.